### ISSUE TITLE
Accessibility issues with code snippets in the docs

### ISSUE THREAD

**Issue by hfroot** [vuelidate/vuelidate#1249]
### Make sure that you are familiar with documentation before submitting an issue.

  https://vuelidate-next.netlify.com/

**Describe the bug**

The documentation contains various different code snippets that are not accessible. This may be introducing accessibility issues into websites relying on the docs.

For the example on the homepage, the problems are:
- `aria-invalid="true"` is not set on the input
- the error message is not linked to the field using `aria-describedby` and `id` of the message(s), or alternatively in the `label` of the field (that also needs to be linked with an `id`)
- consider adding `role="alert" aria-live="polite"` to a div that encompasses the error messages in order for users to be alerted to the new messages on the page
- the error messages are surrounded by `div` tags, rather than a semantic tag such as `p`

https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#input-assistance

On this page, the examples have the same problems. 

**Reproduction URL**

https://vuelidate-next.netlify.app/ : example at the end.
https://vuelidate-next.netlify.app/examples.html

**To Reproduce**

Examine the code examples on the links above.

**Expected behavior**

Code examples to follow good accessibility practices or for there to be accessibility warnings in the site.

**Screenshots**

![Screenshot 2024-07-22 at 17-41-10 vuelidate_packages_docs_src_index md at 08aa51495a4590c17d407a56cbbada9cf370c5d7 · vuelidate_vuelidate](https://github.com/user-attachments/assets/b34ac4d3-65a3-4311-9340-93fbc9ff7ebc)
![Screenshot 2024-07-22 at 17-41-26 Examples Vuelidate](https://github.com/user-attachments/assets/045c0fcf-5f92-49c5-b54c-c88628717be3)

**Additional context**

It would be great to add a page about accessibility considerations of form validation to the site. I wasn't sure whether to open a feature request at the same time though.

I am not an accessibility expert, so take my suggestions with a grain of salt.
